src/components/ColorPalette/ColorPalette.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { colorList } from 'constants';
+import "./ColorPalette.css";
+const ColorPalette = () => {
+    const [colorListStyle, setColorListStyle] = useState("color-list-none")
+    const toggleColorListStyle = () => {
+        setColorListStyle(classDetails => classDetails === "color-list flex-row" ? setColorListStyle("color-list-none") : ("color-list flex-row"))
```

Comment:
What is this? Setting value inside another set?🤔

Reply:
Yes missed it !! Thanks for pointing it out!!